src/Reader/Reader.php

```diff
@@ -89,6 +89,23 @@ class Reader implements ReaderImportInterface
         ],
     ];
 
+    /**
+     * Disable the ability to load external XML entities based on libxml version
+     * 
+     * If we are using libxml < 2.9, unsafe XML entity loading must be
+     * disabled with a flag.
+     * 
+     * If we are using libxml >= 2.9, XML entity loading is disabled by default.
+     * 
+     * @return bool
+     */
+    public static function disableEntityLoader($flag = true)
+    {
+        if (LIBXML_VERSION < 20900) {
+            return libxml_disable_entity_loader($flag);
+        }
```

Comment:
please add `return false` as fallback if `LIBXML_VERSION >= 20900`

```suggestion
        }
        
        return false;
```

Reply:
@samsonasik 
With version 2.9 or higher the entity loading is disabled by default. So `false` is wrong here. Return the value of `$flag` instead.

Reply:
Oh yeah, return `$flag` instead 👍

Reply:
Ok, thanks for feedback, done.